couler/argo_submitter.py

```diff
@@ -33,25 +34,40 @@ def __init__(
         context=None,
         client_configuration=None,
         persist_config=True,
+        # TODO: Remove this once we are done migrating the implementation.
+        go_impl=False,
```

Comment:
Nothing on top of my head. However, we can perhaps change this to an environment variable instead of adding a user-level argument here that will be deprecated/removed eventually. WDYT?

Reply:
Just switched to use env var.